src/bin/vip-wp.js

```diff
@@ -94,6 +112,10 @@ const launchCommandAndGetStreams = async ( { guid, inputToken } ) => {
 		console.log( 'There was an error with the authentication:', err.message );
 	} );
 
+	socket.on( 'disconnect', err => {
+		console.log( 'lost connection' );
```

Comment:
We may want to differentiate between a dropped connection and a command that ended. Currently this will appear after every successful command completes.

Reply:
We should also send these to stderr (console.error) so they don't get streamed to the command output. If someone is running an export, we don't want to inject our error messages into their export file.

Reply:
those were added for debug purpose, but as you suggested we should log to the stderr, and have a differentiation between disconnect because the command is done, and disconnect because a connection is dropped or something.

Looking at this [list of events](https://stackoverflow.com/questions/24224287/list-of-socket-io-events), it looks like we can use `connect_error`.

What do you think @dchymko ?

Reply:
I ended up using `reconnect_attempt` with a `There was an error connecting to the server. Retrying...` message in the stderr